src/legacy/assets/img/camera.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="13" viewBox="0 0 14 13" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Why to place icons to legacy?

Reply:
@CrazyOptimist , I think this came from other's work.

Reply:
Please copy it into our new icon directory, your work will broken later otherwise, because legacy folder will be removed after all.

Reply:
Also the icon would be good to be added to storybook